### PR TITLE
gh-90879: Fix missing parameter for put_nowait()

### PR DESCRIPTION
**Pull request by slateny**
See https://github.com/python/cpython/blob/3.10/Lib/queue.py#L304

https://github.com/python/cpython/issues/90879

### CODE REVIEW

File: Lib/queue.py [hunk 1]
```diff
@@ -298,7 +298,7 @@ def get(self, block=True, timeout=None):
     def put_nowait(self, item):
         '''Put an item into the queue without blocking.
 
-        This is exactly equivalent to `put(item)` and is only provided
+        This is exactly equivalent to `put(item, False)` and is only provided
```
Comment (corona10) [inline Lib/queue.py]:
```suggestion
        This is exactly equivalent to `put(item, block=False)` and is only provided
```

File: Doc/library/queue.rst [hunk 2]
```diff
@@ -248,7 +248,7 @@ SimpleQueue Objects
 
 .. method:: SimpleQueue.put_nowait(item)
 
-   Equivalent to ``put(item)``, provided for compatibility with
+   Equivalent to ``put(item, False)``, provided for compatibility with
```
Comment (corona10) [inline Doc/library/queue.rst]:
```suggestion
   Equivalent to ``put(item, block=False)``, provided for compatibility with
```

**Review by corona10 — CHANGES_REQUESTED** [review]
I would like to suggest changing into more explicit explanation :)

**Comment by slateny** [comment]
I have made the requested changes; please review again

**Comment by slateny** [comment]
(also updated [this](https://github.com/python/cpython/pull/91514/files#diff-5f8aec8fa45976df3a34d74ef406e3c373a0e02a978dd5d8c8d824a6783eafefL141), which didn't have the parameter)

**Review by corona10 — APPROVED** [review]
lgtm

**Pull request merged**